packages/cli/src/commands/deploy.ts

```diff
@@ -18,10 +13,10 @@ export type FieldType = { id: number; name: string; body: string }
 export type DeployArgs = {
   skipPrompts: boolean
   dir: string
-  name?: string
-  token?: string
-  output?: string
-  dotEnvPath?: string
+  name: undefined | string
+  token: undefined | string
+  output: undefined | string
+  dotEnvPath: undefined | string
```

Comment:
@eunjae-lee I'm making these parameters required so that it's less likely to forget to add all arguments.